### PR TITLE
Upgrades compilation

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team, 

This aplies #32, #42,  #43, #46, #48, #49, #53, #54, #56, #57, #61, #62, #63, #64 and also upgrade the project to .NET 5.0 and upgrade node depencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "private": true,
   "dependencies": {
```
Comment (ms-darianbenito) [inline package.json:4]:
```suggestion
  "devDependencies": {
```
A little thing. I known that It is not the PR to made this change. We have this suggestion in [doppler-email-address-validator](https://github.com/MakingSense/doppler-email-address-validator/blob/main/package.json#L5).

Comment (andresmoschini) [reply package.json:4]: I think that I can add this change to this PR, thanks!

File: Dockerfile [hunk 2]
```diff
@@ -31,7 +31,7 @@ RUN dotnet test
 FROM build AS publish
 RUN dotnet publish "DopplerCustomDomain/DopplerCustomDomain.csproj" -c Release -o /app/publish
 
-FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim AS final
+FROM mcr.microsoft.com/dotnet/aspnet:5.0-buster-slim AS final
```
Comment (ms-darianbenito) [inline Dockerfile:34]: I known that this is not the line for my suggestion. Do you need to expose 443 port ?
image

File: package.json (hunk 1 above)
Comment (andresmoschini) [reply package.json:4]: Fixed in #65

File: Dockerfile (hunk 2 above)
Comment (andresmoschini) [reply Dockerfile:34]: Fixed in #66

**Pull request merged**